front/migrations/20230413_runs.ts

```diff
@@ -71,7 +71,7 @@ async function main() {
         if (!app) {
           console.warn(`No app found for project ${projectId}`);
         }
-        // @ts-expect-error
+        // @ts-expect-error old code - no idea what's going on here. But userId does not exists on app.
         const userId = app?.userId;
```

Comment:
@spolu you told me to just ignore the code in these migration files if I understand correctly, but it looks seriously broken.  Do you confirm we should just ignore those?

Reply:
This is a backfilling script. Back when this was written and ran, apps did have a `userId` (it predates the migration to workspaces). 
Maybe we don't need to keep those in the repo forever, WDYT @spolu  ?

Reply:
I'm in favor of keeping them just for context that may be useful in the future. Let's remove the extra comments :+1: 

Since this is a migration that depended on the state of the codebase in the past this is indeed expected and fine to be completely broken.

Reply:
ok I will just trust you blindly.
I need to add a comment for TS to not complain. Otherwise TS asks me for a comment. Hence my comment :)

Reply:
Agreed! But let's try to make the comment as useful and precise as possible eg:

```
// @ts-expert-error old migration code kept for reference
```